frontend/src/components/common/HeaderWrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Box, Flex } from "@chakra-ui/react";
+
+import HomeButton from "./HomeButton";
+
+interface HeaderWrapperProps {
+  content: React.ReactElement;
+}
+
+const HeaderWrapper = ({ content }: HeaderWrapperProps): React.ReactElement => {
```

Comment:
instead of taking the `content` in as props, you can just use React `children`! there are existing examples in the codebase on how to implement this. you can checkout `QuestionSummary.tsx` and `AssessmentInfo.tsx` for one such example.

it's basically what you are implementing here, except it let's you wrap your the content components (it'll make more sense if you look at the codebase).

Reply:
ahhhh okie thanks!!!